### PR TITLE
fix: total price issues

### LINKED ISSUE #13: fixing total price card
Fixed total price detail card in cartPage, added reducers functions for total price calculation.

### PR DESCRIPTION
**Pull request by RahulSingh9131**
This branch contains refactoring of previous code.

### CODE REVIEW

File: src/components/CartCard.js [hunk 1]
```diff
@@ -44,17 +44,17 @@ function CartCard({_id,src,title,categoryName,originalPrice,price,discount,count
                     </div>
                     <div className="quantity-text flex align-center">
                         <p>Quantity :</p>
-                        <button className="quantity-btn flex justify-center align-center" onClick={()=>cartDispatch({type:"INCREMENT_QUANTITY",payload:{_id:_id,price:Number(price),discount:Number(originalPrice)}})}>+</button>
+                        <button className="quantity-btn flex justify-center align-center" onClick={()=>cartDispatch({type:"INCREMENT_QUANTITY",payload:{_id:_id}})}>+</button>
                         <small>{count}</small>
```
Comment (raman1999) [inline src/components/CartCard.js:48]: why _id as object giving directly can send id in payload

**Review by raman1999 — APPROVED** [review]
Code looks clean, good naming conventions done

File: src/components/CartCard.js (hunk 1 above)
Comment (RahulSingh9131) [reply src/components/CartCard.js:48]: actually i was sending few more things in payload too.

**Review by shuntzhere — APPROVED** [review]
Clean code, don't need any changes.

**Pull request merged**